user/build-environment-updates/2017-11-08.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Build Environment Update History
+layout: en
+permalink: /user/build-environment-updates/2017-11-08/
+category: build_env_updates
+date: 2017-11-08
+---
+
+## 2017-11-08
+
+This update applies to macos10.12-xcode9.1 image, for `os: osx` builds
```

Comment:
Maybe we should say that it applies to `osx_image: xcode9.1` builds? I think that `os: osx` will route to the default image.

Reply:
We should perhaps mention the Xcode version that was installed i.e Xcode 9.1 (build 9B55)